v7/notebook_shortcode/notebook_shortcode.py

```diff
@@ -52,8 +52,8 @@ def render_notebook(self, filename, site=None, data=None, lang=None, post=None):
         export_html = HTMLExporter(config=c)
         (notebook_raw, _) = export_html.from_filename(filename)
 
-        # The raw HTML contains garbage (scripts and styles), we can’t leave it in
+        # The raw HTML contains garbage (scripts and styles). Extract only div id=notebook-container and children
```

Comment:
`id` won’t be nice either, I’d prefer to avoid having duplicate `id` tags if someone uses multiple notebooks in a post. But it seems you can’t fix it any better.